### ISSUE TITLE
It crashes and it doesn't display on microsoft store despite having installed it

### ISSUE THREAD

**Issue by JeepHu** [Difegue/Stylophone#77]
As per title, it doesn't display on microsoft store, useful for checking updates. I'm on Windows 10.
It usually crashes after a while after opening it.
I've tracked it with Procmon but I'm not good finding the issue
[Crash Stylophone.zip](https://github.com/Difegue/Stylophone/files/12599077/Crash.Stylophone.zip)
Here is the log in case it should be useful

**Comment by Difegue** [comment]
Have you tried uninstalling and reinstalling it from the store to fix the visibility issue? 
I'm afraid I don't see anything wrong with that procmon log. I should have an update available in a few weeks with some fixes for generic crashes reported by telemetry, maybe yours is in it? 🤷

**Comment by JeepHu** [comment]
Hi Difegue, thanks for checking the log and responding.
Yes I tried uninstalling and reinstalling more than once with no success.
Great to know there will be an update.
No, my crash dump is not there, I usually disable the telemetry. But should another way be there to help you track it down, I'd be pleased to send you

**Comment by Difegue** [comment]
I'd recommend enabling the telemetry in app settings even if you just disable it again postcrash, that'd be helpful 🙏  
![image](https://github.com/Difegue/Stylophone/assets/8237712/ed45e24a-31fe-44b3-aee7-eeda5ee17c13)

**Comment by JeepHu** [comment]
Sorry I had not understood and I thouught you meant the Microsoft telemetry.
I forgot the app had the telemetry. I will reinable it.
In the meantime, I had to reinstall Windows from a backup because of many problems. I'll check wheater the app was affected by them and will get back to you

**Comment by JeepHu** [comment]
Hi Difegue.
The problem was the store and windows search app were corrupted. Thus it's not Stylophone issue, I'm sorry.
Anyway, I just bought and reinstalled it and it shows in the store.
I'll let you know how it is going.
Thank you

**Comment by JeepHu** [comment]
Still some crash:
Nome registro: Application
Origine:       Application Error
Data:          16/09/2023 16:54:16
ID evento:     1000
Categoria attività:(100)
Livello:       Errore
Parole chiave: Classico
Utente:        N/D
Computer:      PC-USER
Descrizione:
Nome dell'applicazione che ha generato l'errore: Stylophone.exe, versione: 0.0.0.0, timestamp: 0x63820fe4
Nome del modulo che ha generato l'errore: KERNELBASE.dll, versione: 10.0.19041.3393, timestamp: 0x6b4de7c9
Codice eccezione: 0xc000027b
Offset errore 0x000000000012d9b2
ID processo che ha generato l'errore: 0xc80
Ora di avvio dell'applicazione che ha generato l'errore: 0x01d9e8ad4ef20d1e
Percorso dell'applicazione che ha generato l'errore: C:\Program Files\WindowsApps\13459Difegue.Stylophone_2.5.6.0_x64__zd7bwy3j4yjfy\Stylophone.exe
Percorso del modulo che ha generato l'errore: C:\WINDOWS\System32\KERNELBASE.dll
ID segnalazione: 32aee96c-e1e5-4a89-b0d1-f274ab61948f
Nome completo pacchetto che ha generato l'errore: 13459Difegue.Stylophone_2.5.6.0_x64__zd7bwy3j4yjfy
ID applicazione relativo al pacchetto che ha generato l'errore: App
XML evento:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" />
    <EventID Qualifiers="0">1000</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>100</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2023-09-16T14:54:16.6940741Z" />
    <EventRecordID>34364</EventRecordID>
    <Correlation />
    <Execution ProcessID="0" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>PC-BOBO</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Stylophone.exe</Data>
    <Data>0.0.0.0</Data>
    <Data>63820fe4</Data>
    <Data>KERNELBASE.dll</Data>
    <Data>10.0.19041.3393</Data>
    <Data>6b4de7c9</Data>
    <Data>c000027b</Data>
    <Data>000000000012d9b2</Data>
    <Data>c80</Data>
    <Data>01d9e8ad4ef20d1e</Data>
    <Data>C:\Program Files\WindowsApps\13459Difegue.Stylophone_2.5.6.0_x64__zd7bwy3j4yjfy\Stylophone.exe</Data>
    <Data>C:\WINDOWS\System32\KERNELBASE.dll</Data>
    <Data>32aee96c-e1e5-4a89-b0d1-f274ab61948f</Data>
    <Data>13459Difegue.Stylophone_2.5.6.0_x64__zd7bwy3j4yjfy</Data>
    <Data>App</Data>
  </EventData>
</Event>

**Comment by Difegue** [comment]
Thank you. I believe it's this crash from AppCenter:  

![image](https://github.com/Difegue/Stylophone/assets/8237712/fcc8774f-c047-445c-bc2c-38b034c7617a)

A fix will be available next release via 596f8c45880ed9007d8766a990076d0e160ab135 .

**Comment by JeepHu** [comment]
I don't know much what that means. I'll wait for the update. Won't it be on the MS Store?
Thank you

**Comment by JeepHu** [comment]
Hi Difegue.
I owe you this: I realised I had a setting in mpd.conf that is not compatible with Stylophone.
I changed this in the configuration file, from:
`default_permissions             ""`
to
`default_permissions             "read,add,control"`
It seems Stylophone commands are not all sent as a priviledged user.
Now it works. The only issue I see now is the queue not being updated when changing song. I need to close the app and open it again for it to update.
Thank you and sorry for my mistake. Though, being able to only use it as a logged user would be great but I guess you have your reasons to develope it this way
